ferretdb_packaging/test_packages/Dockerfile_test_install_deb_packages

```diff
@@ -13,7 +13,8 @@ RUN apt-get install -y --no-install-recommends \
     gnupg2 \
     lsb-release \
     ca-certificates \
-    locales
+    locales \
+    python3
```

Comment:
deb12 (debian:bookworm) building image was failing due to needing `python3` to for diff tool 
for running tests https://github.com/FerretDB/documentdb/blob/main/pg_documentdb/src/test/regress/bin/diff-filter.

For other debian build like deb11 (bullseye), python3 is installed by lsb-release, see https://packages.debian.org/bullseye/lsb-release.

While deb12 (bookworm)'s lsb-release does not need installing python3, see https://manpages.debian.org/bookworm/lsb-release/lsb_release.1.en.html